dart/lib/src/sentry_options.dart

```diff
@@ -70,6 +70,24 @@ class SentryOptions {
     _maxAttachmentSize = maxAttachmentSize;
   }
 
+  /// Maximum number of spans that can be attached to single transaction.
+  ///
+  /// The is an experimental feature. Use at your own risk.
+  int _maxSpans = 1000;
```

Comment:
l: Although this is an experimental feature, `maxSpan` is already being enforced in `SentryTracer`.
Shouldn't we have an option to have it unlimited? Maybe a nullable int, or 0 means unlimited.

I know that 1000 spans is a lot, but I think this would be considered a breaking change.

Reply:
I see your point, i went with the same value as the Java SDK. Maybe @marandaneto or @bruno-garcia could give us more context as to why this value was choosen?

Reply:
It was a missing feature and unlimited is what we don't want, envelopes have a hard limit on size anyway, one can still do 99999999.